ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -565,6 +581,7 @@ PrjFS_Result PrjFS_DeleteFile(
     struct stat path_stat;
     if (0 != stat(fullPath, &path_stat))
     {
+        LogWarning("PrjFS_DeleteFile: stat call failed fullPath=%s errno=%d strerror=%s", fullPath, errno, strerror(errno));
```

Comment:
This is not always an error or unexpected, and so we should not always log at this point.  

In the case of `ENOENT` and `ENOTDIR` we return `PrjFS_Result_Success` (see below).